com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingWindow.cs

```diff
@@ -464,19 +635,37 @@ void DrawBakeButton()
 
         void BakeLightingForSet(ProbeVolumeSceneData.BakingSet set)
         {
-            // Save current scenes:
-            if (!EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo())
-            {
-                Debug.LogError("Can't bake while a scene is dirty!");
-                return;
-            }
-
             var scenesToRestore = new List<string>();
             for (int i = 0; i < EditorSceneManager.sceneCount; i++)
                 scenesToRestore.Add(EditorSceneManager.GetSceneAt(i).path);
 
-            // First, load all the scenes
-            LoadScenesInBakingSet(set);
+            bool sceneSetChanged = scenesToRestore.Count != set.sceneGUIDs.Count;
```

Comment:
What happens if I am baking a different set with the same count of scenes of the ones that I am trying to unload?

Reply:
theres a loop just below that compares the scenes, it just to avoid the loop in case the count is the same